### PR TITLE
Add patches ima-no-cache-v3

### PR DESCRIPTION
**Pull request by alban**

### CODE REVIEW

File: patches/ima-no-cache-v3/0000-cover-letter.patch [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Date: Fri, 19 Jan 2018 12:40:11 +0100
+Subject: [RFC PATCH v3 0/2] ima,fuse: introduce new fs flag FS_IMA_NO_CACHE
+
+This patchset v3 introduces a new fs flag FS_IMA_NO_CACHE and uses it in
+FUSE. This forces files to be re-measured, re-appraised and re-audited
+on file systems with the feature flag FS_IMA_NO_CACHE. In that way,
+cached integrity results won't be used.
+
+There was a previous attempt (unmerged) with a IMA option named "force" and using
+that option for FUSE filesystems. This patches use a different approach
```
Comment (dongsupark) [inline patches/ima-no-cache-v3/0000-cover-letter.patch]: Either `These patches use` or `This patchset uses`?

**Review by dongsupark — APPROVED** [review]
In general LGTM.
Earlier today I already tested, and it worked.
I'm not sure about how the patchset is split, but that's no big deal. It's already ok.

File: patches/ima-no-cache-v3/0000-cover-letter.patch (hunk 1 above)
Comment (alban) [reply patches/ima-no-cache-v3/0000-cover-letter.patch]: updated

Comment (dongsupark) [reply patches/ima-no-cache-v3/0000-cover-letter.patch]: Looks good!

**Comment by dongsupark** [comment]
Ah, I was not aware of this PR staying open. I'm going to merge it. 
We already sent this patchset to LKML in the last week.

**Pull request merged**